### ISSUE TITLE
text.boss is not recognized

### ISSUE THREAD

**Issue by DBTDerpbox** [MehVahdJukaar/polytone#71]
`text.boss` is an option that changes the colors of the bossbars form OptiFine, I can't get it to work (or find it in PolyTone's docs) so I assume the mod doesn't support this feature

Example usage: `text.boss=ebebeb`

Example pack: [PolyTone-bosstext-test.zip](https://github.com/user-attachments/files/16402047/PolyTone-bosstext-test.zip)

**Comment by MehVahdJukaar** [comment]
yeah it doesnt allow changing boss color. its not written on the wiki

**Comment by DBTDerpbox** [comment]
To clarify, do you plan on adding this in the future?

**Comment by MehVahdJukaar** [comment]
not really. There isnt just 1 text color for boss bar. Each bar color uses its respective chat color for its coloring
